### ISSUE TITLE
Terminal crashes when opening Settings

### ISSUE THREAD

**Issue by GoodBoyNeon** [microsoft/terminal#13610]
### Windows Terminal version

1.14.1962.0

### Windows build number

10.0.19044.1826

### Steps to reproduce

Open Windows Terminal
Either press `Ctrl + ,` or open Settings from the dropdown menu

### Expected Behavior

Expected to open the Settings Tab

### Actual Behavior

Windows terminal crashed, sometimes it became unresponsive and crashed after a few seconds

Extra note: Opening settings(JSON) works fine through command palette

**Comment by zadjii-msft** [comment]
Are you running on an ARM PC by any chance? A Feedback Hub trace of the crash might be helpful: /feedback

**Comment by GoodBoyNeon** [comment]
https://aka.ms/AAhkd1t That's the link of the feedback. Thanks for your support!

**Comment by zadjii-msft** [comment]
Did you actually capture a recording of the crash with Feedback Hub? I'm not seeing any dumps in the backend, but it's also entirely possible that they just haven't had the time to fully propagate yet...

**Comment by zadjii-msft** [comment]
... Yea okay there comes some more data. Looks like the backend is just chugging a little slowly today.

**Comment by zadjii-msft** [comment]
Hmm. The FH dump doesn't have a terminal dump in it, but it does link to an exception in MTSM.dll, `ExceptionCode=3221225477, ExceptionOffset=164756`. `3221225477` is `ERROR_ACCESS_DENIED`, which is peculiar. Curious that the blame was on MTSM.dll, and not MTSE.dll.

**Comment by zadjii-msft** [comment]
By any chance was the settings.json file open in another editor when you tried this? Or did you save the file while the settings UI was open/? Just feeling bits of #13673 here.

**Comment by GoodBoyNeon** [comment]
No, I wasn't opening the settings.json while trying that. I am not sure about if I saved the json file while the UI was open. I tried re installing though but it still crashes

**Comment by TJSC1213** [comment]
I have the same problem at all,and the version of my terminal on my windows10 is 10.0.19044.1889

**Comment by zadjii-msft** [comment]
@TJSC1213 can you submit feedback in the same way [I requested above](https://github.com/microsoft/terminal/issues/13610#issuecomment-1196586320)? More data might help ID the root cause here.

**Comment by salastro** [comment]
I have the exact same problem.

**Comment by thals1992** [comment]
I found that I have the same issue. I tried resetting the app as well as uninstalling the app via settings and reinstalled it from the Microsoft Store. It's running on Win 10 22H2, but the 32-bit version on my Dell Venue 8 Pro.

**Comment by zadjii-msft** [comment]
Can you submit feedback in the same way [I requested above](https://github.com/microsoft/terminal/issues/13610#issuecomment-1196586320)? More data might help ID the root cause here.

**Comment by shawn99452** [comment]
This occurs on my Windows 10 22H2 32-bit PC also, running Windows Terminal 1.15.3465.0.  Changes made in settings.json externally take effect in Terminal, so it's reading the file ok, but any attempt to open the Settings UI from inside Terminal causes it to freeze for a second, close the dropdown menu, freeze a few more seconds, and then exit Terminal entirely.  Renaming the settings.json file causes it to make a new one on startup, but the crash when opening the Settings UI still occurs with the default settings.

**Comment by zadjii-msft** [comment]
> Can you submit feedback in the same way [I requested above](https://github.com/microsoft/terminal/issues/13610#issuecomment-1196586320)? More data might help ID the root cause here.

**Comment by shawn99452** [comment]
> > Can you submit feedback in the same way [I requested above](https://github.com/microsoft/terminal/issues/13610#issuecomment-1196586320)? More data might help ID the root cause here.

Trying to, but the Feedback Hub app just shows a blank screen with spinning dots forever when I click "Report a Problem".  Every page in Feedback Hub except the 'Feedback' page works fine :(.  I'll edit my original note, adding the link, if I get it to work.  Is there another way to get the requested data, outside Feedback Hub?

**Comment by zadjii-msft** [comment]
Wow what fortuitous timing that I just wrote up: [Capturing and sending dumps](https://github.com/microsoft/terminal/wiki/Toubleshooting-Tips#capturing-and-sending-dumps)

**Comment by zadjii-msft** [comment]
Hmm, this is the one that I thought might be related to #15243 but ultimately the two are so different, I can't imagine they've got the same root cause. 

It does seem like it's a consistent theme that you're all on 32-bit Windows. That's definitely curious. I wonder if I can even get this to repro in a 32-bit VM on a 64bit

**Comment by zadjii-msft** [comment]
Well, the dump thinks L1139 is the failure here:

https://github.com/microsoft/terminal/blob/dec420eb422a0cac51ab5b1d02ad73b99db4d2bb/src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp#L1122-L1145


I'm pretty confident we're in "wacky undefined compiler bug with coroutines and x86" territory here. 

There's also _no_ dumps of this on the backend, which is curious.

**Comment by jamespack** [comment]
> I wonder if I can even get this to repro in a 32-bit VM on a 64bit

For poop and laughter I tried running in x86 on x64 Windows 11 and you cant. The process dies early on purpose because of driver reqs for conpty.
